devops/providers/azure-devops/templates/infrastructure/scripts/tf-generate-plan.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+. ./commons.sh --source-only
+
+cd $TF_TEMPLATE_WORKING_DIR
+
+# Clear all previous plans
+rm -f *.out
+# Setting the scripts to be run as executable
+chmod 752 *.sh
```

Comment:
This fails if there are no `*.sh` files (i.e., az-hello-world). You can fix it by using something like this:
`chmod -f 752 *.sh || true`. The `-f` silences outputs.